images/atomic-openshift-cluster-autoscaler.yml

```diff
@@ -13,6 +13,7 @@ distgit:
   namespace: containers
 enabled_repos:
 - rhel-server-rpms
+for_payload: 'true'
```

Comment:
this is the string `true` not the boolean `true` isn't it? pretty sure that's not going to be what we want.